src/oss-find-squats/FindSquatsTool.cs

```diff
@@ -0,0 +1,145 @@
+﻿using CommandLine;
+using CommandLine.Text;
+using Microsoft.CodeAnalysis.Sarif;
+using Microsoft.CST.OpenSource.Shared;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Reflection;
+using System.Threading.Tasks;
+using Microsoft.CST.OpenSource.FindSquats;
+
+namespace Microsoft.CST.OpenSource
+{
+    public class FindSquatsTool : OSSGadget
+    {
+        /// <summary>
+        ///     Name of this tool.
+        /// </summary>
+        private const string TOOL_NAME = "oss-find-squats";
+
+        /// <summary>
+        ///     Holds the version string, from the assembly.
+        /// </summary>
+        private static readonly string VERSION = typeof(FindSquatsTool).Assembly?.GetName().Version?.ToString() ?? string.Empty;
+
+        Generative gen { get; set; }
+
+        public class Options
+        {
+            [Usage()]
+            public static IEnumerable<Example> Examples
+            {
+                get
+                {
+                    return new List<Example>() {
+                        new Example("Find Squat Candidates for the Given Packages",
+                        new Options { Targets = new List<string>() {"[options]", "package-urls..." } })};
+                }
+            }
+
+            [Option('o', "output", Required = true, HelpText = "The filename to output.")]
+            public string? OutputFileName { get; set; }
+
+            [Value(0, Required = true,
```

Comment:
the help message format is a bit different from other tools, because the 'hidden' argument is missing here. I hide it in the other tools since its a bit confusing..
>>  value pos. 0    Required. PackgeURL(s) specifier to analyze (required) 
The usage string already has package-urls, and there is a detailed explanation of what a package url is below this message, so it might make sense to make this hidden.

Reply:
Ok